### DISCUSSION TITLE
Moving from 1.54.3 -> 1.56.1 only closest light is shown?

### DISCUSSION THREAD

**Discussion by tcadanklefsen** [google/filament#8289] [Q&A]
Hello,

Moving from 1.54.3->1.56.1 seems to have changed drawing to only show the closest light. Here's where I was at previously:

https://github.com/user-attachments/assets/3bf01661-0bf1-4806-81fb-4ae45085aa04

and here's where I'm at now:

https://github.com/user-attachments/assets/2c26e692-910a-4ae1-8637-57054c04ae34

Any thoughts on what might have changed and where I could start investigating?

**Comment by tcadanklefsen** [comment]
1.54.5 works, 1.55.0 does not - continuing to look.

**Reply by tcadanklefsen** [reply]
@poweifeng I can make this into an issue if you think it is. Might be a 'different way of handling lights' with your descriptor changes in 1.55.0 though? Any advice or thoughts?

**Reply by poweifeng** [reply]
Are you able to create a small sample to reproduce this problem?  This might be an actual bug due to our descrioptor set change, but we need to be able to repro to address it.  Thanks.

**Reply by poweifeng** [reply]
I converted this into a bug #8296.  Let's close this discussion and continue there.
